api.md

```diff
@@ -2654,66 +2656,142 @@ SELECT * FROM timescaledb_information.data_node;
 (2 rows)
 ```
 
-## timescaledb_information.hypertable [](timescaledb_information-hypertable)
+## timescaledb_information.hypertables [](timescaledb_information-hypertables)
 
-Get information about hypertables. If the hypertable is distributed, the
-hypertable statistics reflect the sum of statistics across all distributed chunks.
+Get metadata information about hypertables.
 
-#### Available Columns [](timescaledb_information-hypertable-available-columns)
+#### Available Columns [](timescaledb_information-hypertables-available-columns)
 
 |Name|Description|
 |---|---|
-| `table_schema` | Schema name of the hypertable. |
-| `table_name` | Table name of the hypertable. |
-| `table_owner` | Owner of the hypertable. |
-| `num_dimensions` | Number of dimensions. |
-| `num_chunks` | Number of chunks. |
-| `table_size` |Disk space used by hypertable |
-| `index_size` |Disk space used by indexes|
-| `toast_size` |Disk space of toast tables|
-| `total_size` |Total disk space used by the specified table, including all indexes and TOAST data|
-| `distributed` | (BOOLEAN) Distributed status of the hypertable |
+| `table_schema` | Schema name of the hypertable |
+| `table_name` | Table name of the hypertable |
+| `table_owner` | Owner of the hypertable |
+| `num_dimensions` | Number of dimensions |
+| `num_chunks` | Number of chunks |
+| `compression_enabled` |Is compression enabled on the hypertable?|
+| `is_distributed` | Is the hypertable distributed?|
+| `replication_factor` | Replication factor for a distributed hypertable|
+| `data_nodes` | Nodes on which hypertable is distributed|
+| `tablespaces` |Tablespaces attached to the hypertable |
 
-#### Sample Usage [](timescaledb_information-hypertable-examples)
+#### Sample Usage [](timescaledb_information-hypertables-examples)
 
-Get information about all hypertables.
+Get information about a hypertable.
 
 ```sql
-SELECT * FROM timescaledb_information.hypertable;
+CREATE TABLE dist_table(time timestamptz, device int, temp float);
+SELECT create_distributed_hypertable('dist_table', 'time', 'device', replication_factor => 2);
 
- table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size | distributed
---------------+------------+-------------+----------------+------------+------------+------------+------------+------------+--------------
- public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB     | t
- public       | devices    | postgres    |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB      | f
-(2 rows)
+SELECT * FROM timescaledb_information.hypertables
+  WHERE table_name = 'dist_table';
+
+ table_schema | table_name |    owner    | num_dimensions | num_chunks | compression_enabled | is_distributed | replication_factor |              data_nodes               | tablespaces
```

Comment:
Maybe switch to `\x` for extended views?

Reply:
fixed.